packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -253,6 +253,7 @@ describe('plugin-meetings', () => {
       getReachabilityResults: sinon.stub().resolves(undefined),
       getReachabilityMetrics: sinon.stub().resolves({}),
       stopReachability: sinon.stub(),
+      isSubnetReachable: sinon.stub().returns(true),
```

Comment:
nitpick: in all the tests the stub returns true, it would be nice to have at least one place returning false